src/acceptance-test/java/de/zalando/aruha/nakadi/repository/db/EventTypeCacheTest.java

```diff
@@ -0,0 +1,169 @@
+package de.zalando.aruha.nakadi.repository.db;
+
+import de.zalando.aruha.nakadi.domain.EventType;
+import de.zalando.aruha.nakadi.domain.EventTypeSchema;
+import de.zalando.aruha.nakadi.exceptions.NoSuchEventTypeException;
+import de.zalando.aruha.nakadi.repository.EventTypeRepository;
+import de.zalando.aruha.nakadi.repository.zookeeper.ZooKeeperHolder;
+import org.apache.curator.RetryPolicy;
+import org.apache.curator.framework.CuratorFramework;
+import org.apache.curator.framework.CuratorFrameworkFactory;
+import org.apache.curator.retry.ExponentialBackoffRetry;
+import org.apache.zookeeper.CreateMode;
+import org.echocat.jomon.runtime.concurrent.RetryForSpecifiedTimeStrategy;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import static junit.framework.Assert.assertNotNull;
+import static junit.framework.Assert.assertNull;
+import static junit.framework.Assert.fail;
+import static org.echocat.jomon.runtime.concurrent.Retryer.executeWithRetry;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+public class EventTypeCacheTest {
+
+    private final EventTypeRepository dbRepo = mock(EventTypeRepository.class);
+    private final CuratorFramework client;
+
+    public EventTypeCacheTest() throws Exception {
+        ZooKeeperHolder zkHolder = new ZooKeeperHolder("127.0.0.1:2181", "", "", 0);
+        zkHolder.init();
+        String connecString = zkHolder.get().getZookeeperClient().getCurrentConnectionString();
+        RetryPolicy retryPolicy = new ExponentialBackoffRetry(1000, 3);
+        CuratorFramework cf = CuratorFrameworkFactory.newClient(connecString, retryPolicy);
```

Comment:
What the lines above are doing is kind of dumb, because I first create a client using `ZooKeeperHolder` just to get the connection string in order to create another client.

So I thought it would be enough to hardcode the connection string. I'll try it out and comment the commit link :+1: